Source/WebInspectorUI/UserInterface/Views/OverrideUserPreferencesPopover.js

```diff
@@ -112,11 +112,39 @@ WI.OverrideUserPreferencesPopover = class OverrideUserPreferencesPopover extends
         let contentElement = document.createElement("div");
         contentElement.classList.add("user-preferences-content");
 
-        let accessibilityHeader = contentElement.appendChild(document.createElement("h1"));
-        accessibilityHeader.textContent = WI.UIString("Accessibility Preferences", "Accessibility Preferences @ User Preferences Overrides", "Header for section with accessibility user preferences.");
+        let appearanceHeader = contentElement.appendChild(document.createElement("h1"));
+        appearanceHeader.textContent = WI.UIString("Appearance", "Appearance @ User Preferences Overrides", "Header for section with appearance user preferences.");
+
+        // COMPATIBILITY (macOS 13.0, iOS 16.0): `PrefersColorScheme` value for `Page.UserPreferenceName` did not exist yet.
+        if (InspectorBackend.Enum.Page?.UserPreferenceName?.PrefersColorScheme) {
+            this._createSelectElement({
+                contentElement,
+                id: "override-prefers-color-scheme",
+                label: WI.UIString("Color scheme", "Color scheme @ User Preferences Overrides", "Label for input to override the preference for color scheme."),
+                preferenceName: InspectorBackend.Enum.Page.UserPreferenceName.PrefersColorScheme,
+                preferenceValues: [InspectorBackend.Enum.Page.UserPreferenceValue.Light, InspectorBackend.Enum.Page.UserPreferenceValue.Dark],
+                defaultValue: WI.CSSManager.UserPreferenceDefaultValue,
+            });
+        }
+        // COMPATIBILITY (macOS 13, iOS 16.0): `Page.setForcedAppearance()` was removed in favor of `Page.overrideUserPreference()`
+        else if (InspectorBackend.hasCommand("Page.setForcedAppearance") && InspectorBackend.Enum.Page.Appearance) {
```

Comment:
No need for `else` since these two conditions will be mutually exclusive (since both protocol changes are happening at the same time). This will also eliminate the oddness from the fact the `else` is on a different line than the previous closing curly bracket.

Reply:
Agreed. Fixed.